src/templates/our-story.twig

```diff
@@ -16,17 +16,32 @@ cafe in the heart of local community, where speciality coffees are savoured in a
         GHC offers a full spectrum of brewing opportunities—from pour overs to syphon pots to finely executed
         espresso beverages. Each cup of coffee is made with love, precision and care.
     </p>
-    <img src="/img/espresso-machine_jya3mn_c_scale,w_443.jpg"
+    <img src="/img/espresso-machine_v89s2i_c_scale,w_320.jpg 320w"
         srcset="
-        /img/espresso-machine_jya3mn_c_scale,w_633.jpg 633w,
-        /img/espresso-machine_jya3mn_c_scale,w_795.jpg 795w,
-        /img/espresso-machine_jya3mn_c_scale,w_1070.jpg 1070w,
-        /img/espresso-machine_jya3mn_c_scale,w_1770.jpg 1770w"
+       /img/espresso-machine_v89s2i_c_scale,w_547.jpg 547w,
+       /img/espresso-machine_v89s2i_c_scale,w_730.jpg 730w,
+       /img/espresso-machine_v89s2i_c_scale,w_887.jpg 887w,
+       /img/espresso-machine_v89s2i_c_scale,w_1030.jpg 1030w,
+       /img/espresso-machine_v89s2i_c_scale,w_1161.jpg 1161w,
+       /img/espresso-machine_v89s2i_c_scale,w_1283.jpg 1283w,
+       /img/espresso-machine_v89s2i_c_scale,w_1396.jpg 1396w,
+       /img/espresso-machine_v89s2i_c_scale,w_1510.jpg 1510w,
+       /img/espresso-machine_v89s2i_c_scale,w_1597.jpg 1597w,
+       /img/espresso-machine_v89s2i_c_scale,w_1689.jpg 1689w,
+       /img/espresso-machine_v89s2i_c_scale,w_1776.jpg 1776w,
+       /img/espresso-machine_v89s2i_c_scale,w_1873.jpg 1873w,
+       /img/espresso-machine_v89s2i_c_scale,w_1951.jpg 1951w,
+       /img/espresso-machine_v89s2i_c_scale,w_2024.jpg 2024w,
+       /img/espresso-machine_v89s2i_c_scale,w_2096.jpg 2096w,
+       /img/espresso-machine_v89s2i_c_scale,w_2166.jpg 2166w,
+       /img/espresso-machine_v89s2i_c_scale,w_2179.jpg 2179w,
+       /img/espresso-machine_v89s2i_c_scale,w_2180.jpg 2180w"
```

Comment:
I'd remove a few
espresso-machine_v89s2i_c_scale,w_2166.jpg is 480348 bytes (should be enough for hiDPI displays) so keep just this
espresso-machine_v89s2i_c_scale,w_2179.jpg is 488857 bytes (just 8 kB more)
espresso-machine_v89s2i_c_scale,w_2180.jpg is 566132 bytes (a lot bigger and one pixel wider…)